src/databricks/labs/remorph/reconcile/recon_capture.py

```diff
@@ -236,6 +236,10 @@ def _apply_threshold(
             if table_conf.table_thresholds
             else []
         )
+        # if not table thresholds are provided return true
+        if not thresholds:
+            return True
+
         res = None
         if thresholds:
```

Comment:
When the `thresholds` are None, we're returning `True`. So, again we don't need to check the `thresholds`.
```suggestion
```

Reply:
done